lib/http/node.js

```diff
@@ -100,6 +100,9 @@ AWS.NodeHttpClient = AWS.util.inherit({
           total: totalBytes
         });
       });
+      if (typeof body === 'string') {
+        body = new Buffer(body);
```

Comment:
Since the Buffer constructors are deprecated, we might want to add some feature detection to try `Buffer.from` first, then `new Buffer`.